Documentation/markdown/SetupHelp.md

```diff
@@ -0,0 +1,86 @@
+# OpenInfraPlatform Installation 
+
+***
+## Prerequisites 
+
+Before you can proceed to the Installation of **OpenInfraPlatform**, check, if these steps are completed:
+
+1. **Visual Studio 2017** is installed to your computer - find [here](https://my.visualstudio.com/Downloads?q=visual%20studio%202017&wt.mc_id=o~msft~vscom~older-downloads)
+2. **CMake 3.17.0** is installed - find [here](https://cmake.org/download/)
+3. **Qt 5.12.1** is installed and included in the path (environment variable)-  find [here](https://www.qt.io/download-open-source)
+
+	*	Download Qt Online Installer 
+	*	In the meantime, your computer is downloading Qt installer, create Qt account. 
+	*	Sign in with your new account to Qt installer and select directory, where Qt will be installed. (** C:\Qt ** should be default option, if not write **this** path out)
+	*	Select components, you want to install:
+
+		* Check the *Archive* box
+		* Select *Filter*
+		* Open section **Qt 5.12.1**
+		* **Mandatory:** Select *binaries x64 msvc2017*
+		* **Mandatory:** Select *mingw*
+		
+		![](./fig/Qt_Installation_settings.png)
+
+4. **Boost 1_65_1** is installed and included in the path (environment variable) -  find [here](https://sourceforge.net/projects/boost/files/boost-binaries/1.65.1/boost_1_65_1-msvc-14.1-64.exe/download)
+5. **Anaconda 2**  - find [here](https://repo.anaconda.com/archive/Anaconda2-2019.10-Windows-x86_64.exe) (For Windows users) 
+	 **NOTE:** - If you are using Linux or MacOS, you will find Anaconda installer archive [here](https://repo.anaconda.com/archive/) (Choose the version with **Python 2.7**(**Anaconda2**))
```

Comment:
Not necessary since we're not supporting Unix.

Reply:
The note came from me. The information 'Python 2.7' was useful, at least during my setup.
@hechth I want to suggest to delete the note and modify point 5 to something like the following. Is this OK?
5. **Anaconda2** (version with Python 2.7) - find [here](https://repo.anaconda.com/archive/Anaconda2-2019.10-Windows-x86_64.exe)

Reply:
Done